MotionPhotoReader/app/src/main/java/com/google/android/libraries/motionphotoreader/XmpParser.java

```diff
@@ -0,0 +1,46 @@
+package com.google.android.libraries.motionphotoreader;
+
+import androidx.annotation.Nullable;
+
+import com.adobe.internal.xmp.XMPException;
+import com.adobe.internal.xmp.XMPMeta;
+import com.adobe.internal.xmp.XMPMetaFactory;
+import com.google.common.io.ByteStreams;
+import com.google.common.primitives.Bytes;
+
+import java.io.ByteArrayOutputStream;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.util.Arrays;
+
+/**
+ * The XmpParser class is a package-private class intended to help extract the microvideo offset
+ * information from the XMP metadata of the given Motion Photo. This class is for internal use only.
```

Comment:
nit: remove the "This class is for internal use only" part since the package private implies that already.